CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 ## [1.4.1]:
 * Update Native Auth logging levels for improved clarity and consistency. #2184
+* Increased macOS minimum version to 10.15 #2220
```

Comment:
Can you move this to a new entry? You can use `## [TBD]` like [this](https://github.com/AzureAD/microsoft-authentication-library-for-objc/blob/a4fa7c2e5cd81bcb1680957c2f1def0081577c08/CHANGELOG.md).

Reply:
Done